packages/react/src/components/Authenticator/Provider/index.tsx

```diff
@@ -23,19 +23,20 @@ const useAuthenticatorValue = ({
   signUpAttributes,
   services,
 }: ProviderProps) => {
-  const [state, send] = useMachine(
-    () =>
-      createAuthenticatorMachine({
+  const [state, send] = useMachine(() => createAuthenticatorMachine());
```

Comment:
In later PR, I'll adjust this so that `machine` can be created without explicit `<Authenticator.Provider>` around package users' app. This will allow `useAuthenticator` to be used outside `<Authenticator >` internals and still have access to global machine.